### ISSUE TITLE
some parameters are not working

### ISSUE THREAD

**Issue by YoungCholKim** [unexge/tbr#2]
--all_time is not working

>  tensorboard-reporter \
>  --run_dir ./models \
>  --interval_hour 1 \
>  --tag "loss" \
>  --slack_channels "#tb-report" \
>  --all_time
usage: tensorboard-reporter [-h] --run_dir RUN_DIR --tag TAG --interval_hour
                            INTERVAL_HOUR --slack_channels SLACK_CHANNELS
tensorboard-reporter: error: unrecognized arguments: --all_time

--interval_hour accepts only integer, 

>  tensorboard-reporter \
>  --run_dir ./models \
>  --interval_hour 0.1 \
>  --tag "loss" \
>  --slack_channels "#tb-report" \
>  --all_time
usage: tensorboard-reporter [-h] --run_dir RUN_DIR --tag TAG --interval_hour
                            INTERVAL_HOUR --slack_channels SLACK_CHANNELS
tensorboard-reporter: error: argument --interval_hour: invalid int value: '0.1'

--tag is optional from readme, but it's required

>  tensorboard-reporter \
>  --run_dir ./t5-models/ \
>  --interval_hour 0 \
>  --slack_channels "#tb-report" \
>  --all_time
usage: tensorboard-reporter [-h] --run_dir RUN_DIR --tag TAG --interval_hour
                            INTERVAL_HOUR --slack_channels SLACK_CHANNELS
tensorboard-reporter: error: the following arguments are required: --tag

even if pass those params,
I'v got only blank images
![image](https://user-images.githubusercontent.com/11783519/96062647-282e2a80-0ed1-11eb-973d-7461e9d7212c.png)

it runs on TensorBoard 2.3.0

**Comment by unexge** [comment]
Hi @YoungCholKim, these are new features [added](https://github.com/unexge/tensorboard-reporter/pull/1) by @FarisHijazi. I forget to update package on PyPi :). Can you try again with version [`0.0.4`](https://pypi.org/project/tensorboard-reporter/0.0.4/)?

**Comment by YoungCholKim** [comment]
after update to 0.0.4, the params are working.

but I've got a message 
`found 0 summaries
`

I doubt it can't access the directory...

Here is my directory to watch : 
![image](https://user-images.githubusercontent.com/11783519/96851712-48db1f00-1493-11eb-8e98-438dca22a276.png)

**Comment by unexge** [comment]
sorry for the late response, can you try again with [`0.0.5`](https://pypi.org/project/tensorboard-reporter/0.0.5/)?

i was able to get reports generated with:
```python
# example from https://www.tensorflow.org/tensorboard/get_started
import tensorflow as tf
import datetime

mnist = tf.keras.datasets.mnist

(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

def create_model():
    return tf.keras.models.Sequential(
        [
            tf.keras.layers.Flatten(input_shape=(28, 28)),
            tf.keras.layers.Dense(512, activation="relu"),
            tf.keras.layers.Dropout(0.2),
            tf.keras.layers.Dense(10, activation="softmax"),
        ]
    )

model = create_model()
model.compile(
    optimizer="adam", loss="sparse_categorical_crossentropy", metrics=["accuracy"]
)

log_dir = "logs/fit/" + datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)

model.fit(
    x=x_train,
    y=y_train,
    epochs=5,
    validation_data=(x_test, y_test),
    callbacks=[tensorboard_callback],
)
```

with this params:
```bash
$ SLACK_BOT_TOKEN="xoxb-abc-1232" tensorboard-reporter \
 --run_dir ./logs --all_time \
 --interval_hour 0.5 --slack_channels "#tensorboard-reports"
found 20 summaries
2 summaries groupted by tags
```
![image](https://user-images.githubusercontent.com/16212576/97609107-3ed19700-1a24-11eb-86bc-68d3b6bdb0df.png)
